testsuite/features/init_clients/min_deblike_salt.feature

```diff
@@ -7,6 +7,9 @@
 @deblike_minion
 Feature: Bootstrap a Debian-like minion and do some basic operations on it
 
+  Scenario: Clean up sumaform leftovers on SLES minion
```

Comment:
`SLES minion`?

Reply:
Arf, right, copy-pasting everywhere :) Will fix in all the places